src/redux/categories/index.js

```diff
@@ -0,0 +1,24 @@
+// categories.js
+
+// Actions
+const CHECK_STATUS = 'CREATE';
```

Comment:
- [ ] **Required:** You are required to use `duck's pattern` in defining the actions of the reducer.